basic-mongo-two/database/index.js

```diff
@@ -0,0 +1,53 @@
+const { MongoClient, ObjectID } = require('mongodb');
+
+const MONGO_URL = 'mongodb://localhost:27017';
```

Comment:
Constants values that can be changed later or depends on different systems should be in environment variables.